components/style/stylesheets/mod.rs

```diff
@@ -116,21 +116,37 @@ impl MallocSizeOfWithGuard for CssRule {
         malloc_size_of: MallocSizeOfFn
     ) -> usize {
         match *self {
+            // Not all fields are currently fully measured. Extra measurement
+            // may be added later.
+
+            CssRule::Namespace(_) => 0,
+            CssRule::Import(_) => 0,
```

Comment:
Add a comment here that we don't need to measure the imported sheet, since we'll find it on the C++ side in the child list of this `ServoStyleSheet`.